.travis.yml

```diff
@@ -10,21 +10,19 @@ env:
   - SAILS_ADAPTER_NAME=postgresql
   - SAILS_ADAPTER_NAME=mysql
 
-before_script:
-  - npm link .
+before_install:
+  - sudo apt-get update -qq
```

Comment:
doesnt this effects the tests speed for new machines? is it worth the optimization?